js/libs/keycloak-js/dist/keycloak.d.ts

```diff
@@ -531,8 +531,10 @@ declare class Keycloak {
 
 	/**
 	* Called when a AIA has been requested by the application.
+	* @param status the outcome of the required action
+	* @param action the name of the required action
```

Comment:
Thanks for adding some docs here, I feel though we can be a bit more explicit and either link to documentation that explains what the possible options are here. Or even better, document these values in-line.

Reply:
I added some example values there. That should be enough for users to figure out what they can expect there.